### PR TITLE
update auth-api.py for EU

### PR DESCRIPTION
**Pull request by ali-shaw**
Related to https://github.com/DataDog/documentation/pull/5846 updating to include `api_host` code example for EU customers

### What does this PR do?
Updates the code example to align with https://github.com/DataDog/documentation/pull/5846
### Motivation
Based on customer feedback - providing a clear example of how to specify EU endpoint for Python

### CODE REVIEW

File: content/en/api/authentication/code_snippets/api-auth.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from datadog import initialize, api
 
 options = {'api_key': '<YOUR_API_KEY>',
-           'app_key': '<YOUR_APP_KEY>'}
+           'app_key': '<YOUR_APP_KEY>',
+           'api_host': 'https://api.datadoghq.eu` #optional}
```
Comment (l0k0ms) [inline content/en/api/authentication/code_snippets/api-auth.py]:
```suggestion
           'api_host': 'https://api.datadoghq.eu`}
```

Comment (l0k0ms) [reply content/en/api/authentication/code_snippets/api-auth.py]: No need to precise that it's optional.

Comment (ali-shaw) [reply content/en/api/authentication/code_snippets/api-auth.py]: thanks @l0k0ms 👍

**Comment by l0k0ms** [comment]
Closing this PR and following it here: https://github.com/DataDog/documentation/pull/5864

**Pull request closed without merging**